third_party/zlib.BUILD

```diff
@@ -0,0 +1,61 @@
+# Copied from https://github.com/protocolbuffers/protobuf/blob/master/third_party/zlib.BUILD
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])  # BSD/MIT-like license (for zlib)
+
+_ZLIB_HEADERS = [
+    "crc32.h",
+    "deflate.h",
+    "gzguts.h",
+    "inffast.h",
+    "inffixed.h",
+    "inflate.h",
+    "inftrees.h",
+    "trees.h",
+    "zconf.h",
+    "zlib.h",
+    "zutil.h",
+]
+
+_ZLIB_PREFIXED_HEADERS = ["zlib/include/" + hdr for hdr in _ZLIB_HEADERS]
+
+# In order to limit the damage from the `includes` propagation
```

Comment:
Did you come up with this, or was it advised somewhere?

Reply:
I think I got this from somewhere else - will search where

Reply:
Oh good job past Marie! Looks like I added it at the top of the file: `# Copied from https://github.com/protocolbuffers/protobuf/blob/master/third_party/zlib.BUILD`

Reply:
I also got some other from here I think: https://github.com/bazelbuild/bazel-federation